### DISCUSSION TITLE
原神进度以及未来规划 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#21] [Announcements]
# 原神进度以及未来规划 | YuZhangWang的领域

更新时间为2023.8.11，以上是经过了187天之后的数据，先定下一个基调，我是成就党，深渊能过就行 任务魔神任务关于魔神任务，我大概花了1个月，过完了全部的剧情。我是使用的英文语音，因此听着磨磨耳朵也好，对于总体的剧情也有所大概了解，当我遇到不懂的地方，就去看影月月的视频 传说任务传说任务，做完了才发现，没有成就可以获得，大部分直接跟着任务提示做就可以了，某部分有隐藏宝箱，因此不确定的话，也

https://yuzhang.wang/129-genshin-progress-planning/

**Comment by NOeSIS-yu** [comment]
卧槽，玩个原神还能写个规划
